console/console-init/ui/src/components/TypeAheadSelect/TypeAheadSelect.tsx

```diff
@@ -79,24 +82,28 @@ const TypeAheadSelect: React.FunctionComponent<ITypeAheadSelectProps> = ({
           if (options && options.length > 0) {
             setOptions(options);
           } else {
-            setOptions([
-              <SelectOption
-                key={0}
-                value={TypeAheadMessage.NO_RESULT_FOUND}
-                isDisabled={true}
-              />
-            ]);
+            !isCreatable &&
+              setOptions([
+                <SelectOption
+                  key={0}
+                  value={TypeAheadMessage.NO_RESULT_FOUND}
+                  isDisabled={true}
+                />
+              ]);
           }
         });
     }
 
-    const options = [
-      <SelectOption
-        value={TypeAheadMessage.MORE_CHAR_REQUIRED}
-        key="1"
-        isDisabled={true}
-      />
-    ];
+    let options: any[] = [];
```

Comment:
Can the type here be SelectOptionObject[]

Reply:
I tried to use SelectOptionObject but it cause the issue. based on the situation internal option will change.